### DISCUSSION TITLE
[Feature Request]: Rotate latent

### DISCUSSION THREAD

**Discussion by guzuligo** [hako-mikan/sd-webui-regional-prompter#318] [Ideas]
When faces are not in normal orientation, stable diffusion tends to mess them up.

I suggest if there is a way to feed a rotated latent to SD depending on the mask. This way, the base prompt or the controlNet will create the rotated face/hand/etc.., while the masks or the ADDCOL will have the face presented in the best suited SD orientation.

Example prompt: ```faces from below. Friends circle```
![image](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/4189008/13370ac8-9771-4609-8dfb-274e8bf20b04)

**Comment by hako-mikan** [comment]
That's an interesting suggestion indeed. Handstands and the like are 
challenging for SD, but it might be possible to achieve by flipping 
Latent in the middle of the steps. Thank you for the proposal.

**Comment by hako-mikan** [comment]
I am conducting some experiments, but they appear to be challenging. Rotating the Latent midway results in images consisting solely of noise. There may be a crucial oversight, or it could fundamentally be unfeasible.

**Reply by guzuligo** [reply]
I hope it is not the later 😅

You might have already tried this, but my suggestion would be to try different samplers, lower CFG, generation mode as Latent instead of Attention, and maybe different base ratio weights.

**Reply by guzuligo** [reply]
I'm not sure if this is helpful, but have you looked at the way tiling was made? It is a standard feature in A1111.
I think they present the latent to the model differently and the model is accepting that new presentation.
I hope there could be a useful trick from there.
